src/platform/user/authentication/utilities.js

```diff
@@ -5,8 +5,19 @@ import recordEvent from '../../monitoring/record-event';
 import { apiRequest } from '../../utilities/api';
 import environment from '../../utilities/environment';
 
+import brandConsolidation from '../../brand-consolidation';
+
+let successRelay = 'vetsgov';
+if (brandConsolidation.isEnabled()) {
+  if (brandConsolidation.isPreview()) {
+    successRelay = 'preview_vagov';
+  } else {
+    successRelay = 'vagov';
+  }
+}
+
 const SESSIONS_URI = `${environment.API_URL}/sessions`;
-const redirectUrl = (type) => `${SESSIONS_URI}/${type}/new`;
+const redirectUrl = (type) => `${SESSIONS_URI}/${type}/new?success_relay=${successRelay}`;
```

Comment:
Compatible, as this defaults to `vetsgov` and has already been implemented in the API ✅ 

cc @omgitsbillryan @saneshark @U-DON